File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        Map<Integer, Integer> map = new HashMap<>();
        int answer[] = new int[2];
        for(int i=0;i<nums.length;i++){
           int diff = target - nums[i];
           if(map.containsKey(diff)){
            answer[1] = i;
            answer[0] = map.get(diff);
            return answer;
           }
           map.put(nums[i], i);
        }
        return answer;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        hashMap = {}
        for a in nums:
            if a not in hashMap:
                hashMap[a] = 0
            hashMap[a] += 1
            
        return [a[0] for a in sorted(hashMap.items(), key=lambda x:x[1], reverse=True)][:k]
        